examples/rate-limiting/main.bal

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerinax/redis;
+
+// Define the Redis client configuration
+configurable string host = ?;
```

Comment:
I think we are following Redis official example [1] as a reference for this sample code. Shall we include the Redis Insight part to this as well ? I think we need to add some minor configs to the docker-compose to make it work. 

[1] - [https://developer.redis.com/howtos/ratelimiting/](https://developer.redis.com/howtos/ratelimiting/)

Reply:
Updated with https://github.com/ballerina-platform/module-ballerinax-redis/pull/220/commits/f50d9b7e3fabd8df6424719a6734a5d8be5c7280